### PR TITLE
#2060 update multiple coin with uint64

### PR DESCRIPTION
**Pull request by ThanhNhann**
Closes: #2060 

## What is the purpose of the change

Add multiplication of sdk.Coins with a number

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? no
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? no
  - How is the feature or change documented? not documented

### CODE REVIEW

File: osmomath/mul_coins_with_number_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		"5 * 1": {
+			sdk.NewInt(5), 1, sdk.NewInt(5),
+		},
+		"5 * 10": {
+			sdk.NewInt(5), 10, sdk.NewInt(50),
+		},
+	}
+
+	for name, tt := range tests {
+		fmt.Println("Start")
```
Comment (mattverse) [inline osmomath/mul_coins_with_number_test.go:30]: TODO : erase this

**Review by mattverse — COMMENTED** [review]
Thanks for this PR! 

Would this PR be able to address some of the places where we can replace existing code with this method? I'm concerned about these methods not getting used and ending up as extra boiler plates

File: osmomath/mul_coins_with_number_test.go (hunk 1 above)
Comment (ThanhNhann) [reply osmomath/mul_coins_with_number_test.go:30]: thanks for your reply. This code can be used for testing the cases relevant to the coin first sir

**Review by mattverse — COMMENTED** [review]
oh, apologies if I was a little unclear. 

I think this implementation itself lgtm, but I was wondering if we could actually replace some existing parts or test cases in the codebase with this new method.( don't want this method to be extra overhead that remains unused)

**Comment by ThanhNhann** [comment]
@mattverse maybe not now, we can close it now and reopen it when need

**Pull request closed without merging**